File: AlternativeQL/src/org/ql/ast/literal/Literal.java
package org.ql.ast.literal;

import org.ql.ast.type.Type;

public class Literal {
    private Type type;

    public Literal() {
    }

    public Type getType() {
        return type;
    }

    public void setType(Type type) {
        this.type = type;
    }
}

File: OffByOne/src/OffByOne.Ql/Ast/Literals/Base/Literal.cs
namespace OffByOne.Ql.Ast.Literals.Base
{
    using System.Collections.Generic;
    using OffByOne.Ql.Ast.Expressions;

    public abstract class Literal : Expression
    {
        public override ISet<string> GetDependencies()
        {
            return new HashSet<string>();
        }
    }
}
